custom/icds_reports/reports/fact_sheets.py

```diff
@@ -623,22 +612,35 @@ def get_data(self):
 
         needed_data_sources = self._get_needed_data_sources(config)
 
-        data_sources = [
-            data_source.get_data()
-            for k, data_source in self.data_sources(self.config).items()
-            if k in needed_data_sources
-        ]
-
-        all_data = self._get_all_data(data_sources)
-
-        sql_location = self.config.get('sql_location')
+        data_sources = list()
```

Comment:
nit, `data_sources = []` is more standard in our code base

Reply:
I see. I just feel `list()` more pythonic :P But sure can make it `[]` after profiling is done